superset-frontend/spec/javascripts/sqllab/QueryTable_spec.jsx

```diff
@@ -35,12 +39,17 @@ describe('QueryTable', () => {
     expect(React.isValidElement(<QueryTable {...mockedProps} />)).toBe(true);
   });
   it('renders a proper table', () => {
-    const wrapper = shallow(<QueryTable {...mockedProps} />);
-    const tableWrapper = wrapper
-      .find(TableView)
-      .shallow()
-      .find(TableCollection)
-      .shallow();
+    const mockStore = configureStore([thunk]);
+    const store = mockStore({});
+    const spyOnUseSelector = jest.spyOn(redux, 'useSelector');
```

Comment:
instead of using a spy for this, can you push the data that you need into the store when you load it with the provider?

Reply:
Ok, I added it to the store, and added user as a fixture.